### PR TITLE
Changes in src of the app ,

### PR DESCRIPTION
**Pull request by Manas-kashyap**
changes like logo of app , done , and a basic layout with all types of
packages called that will be needed for it , and a basic layout is ready

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,5 @@
 - Include the screenshot of the App while submitting your PR.
 
 ## Contributors 
+
+- Manas kashyap
```
Comment (thisisayush) [inline README.md:20]: You need not add it yourself. Will be added automatically.

File: src/app/src/androidTest/java/com/example/manas/alias/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package in.asetalias.alias;
+package com.example.manas.alias;
```
Comment (thisisayush) [inline src/app/src/androidTest/java/com/example/manas/alias/ExampleInstrumentedTest.java:1]: Incorrect Package name.

File: src/app/src/androidTest/java/com/example/manas/alias/ExampleInstrumentedTest.java [hunk 3]
```diff
@@ -21,6 +21,6 @@ public void useAppContext() throws Exception {
         // Context of the app under test.
         Context appContext = InstrumentationRegistry.getTargetContext();
 
-        assertEquals("in.asetalias.alias", appContext.getPackageName());
+        assertEquals("com.example.manas.alias", appContext.getPackageName());
```
Comment (thisisayush) [inline src/app/src/androidTest/java/com/example/manas/alias/ExampleInstrumentedTest.java:24]: Incorrect
Replace com.example.manas with in.asetalias

File: src/app/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -1,17 +1,25 @@
...
-
+    package="com.example.manas.alias">
+    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.READ_PROFILE" />
+    <uses-permission android:name="android.permission.READ_CONTACTS" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+    <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```
Comment (thisisayush) [inline src/app/src/main/AndroidManifest.xml:11]: Explain the use of above permissions.

File: src/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -1,17 +1,25 @@
...
         android:label="@string/app_name"
+        android:roundIcon="@mipmap/asetalias"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity
-            android:name=".MainActivity"
-            android:label="@string/app_name"
-            android:theme="@style/AppTheme.NoActionBar">
+        <activity android:name=".MainActivity"
+            android:label="ALiAS"
```
Comment (thisisayush) [inline src/app/src/main/AndroidManifest.xml:20]: Constants to be avoided. Use ````@string``` instead

File: src/app/src/main/java/com/example/manas/alias/MainActivity.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+import android.support.v7.app.AppCompatActivity;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+}
```
Comment (thisisayush) [inline src/app/src/main/java/com/example/manas/alias/MainActivity.java:13]: Wrong file. java/com/example/manas shouldn;'t exist. Edit the main file instead.

File: src/app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -1,32 +1,20 @@
...
-    tools:context="in.asetalias.alias.MainActivity">
+    tools:context="com.example.manas.alias.MainActivity">
 
-    <android.support.design.widget.AppBarLayout
-        android:layout_width="match_parent"
+    <TextView
+        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:theme="@style/AppTheme.AppBarOverlay">
+        android:text="WELCOME TO ALIAS APP"
```
Comment (thisisayush) [inline src/app/src/main/res/layout/activity_main.xml:12]: Use ```@string``` instead

File: src/app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -1,32 +1,20 @@
...
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:text="ALiAS"
-                android:textAppearance="@style/TextAppearance.AppCompat.Title"
-                android:textAlignment="center" />
-        </android.support.v7.widget.Toolbar>
-    </android.support.design.widget.AppBarLayout>
-
-    <include layout="@layout/content_main" />
-
```
Comment (thisisayush) [inline src/app/src/main/res/layout/activity_main.xml:31]: Bring back the AppBar and use java code to hide it on not required places.

File: src/app/src/test/java/com/example/manas/alias/ExampleUnitTest.java [hunk 9]
```diff
@@ -1,4 +1,4 @@
-package in.asetalias.alias;
+package com.example.manas.alias;
```
Comment (thisisayush) [inline src/app/src/test/java/com/example/manas/alias/ExampleUnitTest.java:1]: Incorrect package

**Review by AniVerma17 — CHANGES_REQUESTED** [review]
Please modify application ID and package names to in.asetalias.alias to avoid any future conflicts.

**Review by thisisayush — CHANGES_REQUESTED** [review]
Too many errors to merge. Please fix the errors and submit the PR after rebasing again.

**Comment by Manas-kashyap** [comment]
ok doing it all again

**Pull request closed without merging**